src/main/kotlin/com/sourcegraph/cody/edit/InstructionsInputTextArea.kt

```diff
@@ -63,7 +63,7 @@ class InstructionsInputTextArea(parentDisposable: Disposable) :
         setRenderingHint(RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON)
         color = EditUtil.getThemeColor("Component.infoForeground")
         val leftMargin = 15
-        drawString(GHOST_TEXT, leftMargin, (fontMetrics.height * 1.5).toInt())
+        drawString(GHOST_TEXT, leftMargin, (fontMetrics.height * 1.5).toInt()-2)
```

Comment:
I think you're going to need to run `./gradlew spotlessApply` in order to fix the formatting before you can merge.

Reply:
Thanks for the tip! Done